src/module/vue/sf-charactersheet.vue

```diff
@@ -55,7 +61,11 @@
       <div class="flexcol margin-right condition-meters">
         <div class="flexrow nogrow" style="flex-wrap: nowrap">
           <!-- TODO: restyle as h4-like -->
-          <btn-rollstat class="vertical-v2 text" :actor="actor" attr="health">
+          <btn-rollstat
+            class="vertical-v2 nogrow text"
+            :actor="actor"
```

Comment:
We should audit these, because I made it so that `actor` and `$actor` go through provide/inject instead of being props that are passed around everywhere. It's fine for now, but just so you're aware.

Reply:
hmm, okay. reckon it's worth opening an issue for that? little easier to keep on the radar that way

Reply:
Done.